lib/chrono_model/adapter/migrations.rb

```diff
@@ -197,6 +201,7 @@ def add_column(table_name, column_name, type, **options)
       # the temporal schema and update the triggers.
       #
       def rename_column(table_name, *)
+        puts "Entering 'rename column'"
```

Comment:
please remove

Reply:
Finally getting back to this, this is fixed :)